docs/sphinx/ome-xml/java-library.rst

```diff
@@ -10,8 +10,9 @@ metadata processing facilities form the backbone of the
 Download
 --------
 
-You can download **ome-xml.jar** from the 
-:bf_downloads:`Bio-Formats downloads <>` page. The
+**ome-xml.jar** is included in **bioformats_package.jar** available to
+download from the :bf_downloads:`Bio-Formats download page <>`. Alternatively,
+you can download **ome-xml.jar** as a single jar from `Maven <http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.openmicroscopy%22%20AND%20a%3A%22ome-xml%22>`_. The
```

Comment:
Interestingly, the anchor was not found in https://ci.openmicroscopy.org/view/Docs/job/MODEL-merge-docs/262/console although the link resolves to me.
I wonder if people will download the JAR manually. If a Java project wants to consume the ome-xml Java library, they will probably use a dependency management tool (Maven, Gradle, Ivy). It might be sufficient to say the artifact is hosted Maven Central (with a link to https://search.maven.org/) under the group `org.openmicroscopy` and the name is `ome-xml`.